core/protocols/aws-json-protocol/src/main/java/software/amazon/awssdk/protocols/json/internal/unmarshall/JsonProtocolUnmarshaller.java

```diff
@@ -103,7 +103,7 @@ private static JsonUnmarshallerRegistry createUnmarshallerRegistry(
                 .build();
     }
 
-    private static SdkBytes unmarshallSdkBytes(JsonUnmarshallerContext context,
+    public static SdkBytes unmarshallSdkBytes(JsonUnmarshallerContext context,
```

Comment:
Hmm why are these changed to `public`?

Reply:
so that they could be used, but I gather from other comments that it's not desirable

Reply:
Have restored this file, and copied relevant code locally instead.